wp/lib/bap_wp/src/precondition.mli

```diff
@@ -364,4 +368,21 @@ val set_sp_range : Env.t -> Constr.t
     all specified registers cannot point to the uninitalized stack region.
     They must be either below the bottom of the stack or above the initial
     stack pointer. *)
+
 val construct_pointer_constraint : Constr.z3_expr list -> Env.t -> (Constr.z3_expr list) option -> Env.t option -> Constr.t
+
+(** [loop_unroll num_unroll] returns a function that will unroll loops
```

Comment:
Maybe be more explicit: "will compute the precondition of nodes within loops by assuming that the loop was traversed at most `num_unroll` times".

Or something. Not too big an issue.